service/utils/file_utils.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+/* eslint-disable
```

Comment:
Can you remove this? I'm thinking of disabling that rule completely since we've made the decision that styles will exist in the component files so we'll have tons of magic numbers everywhere.